tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -107,23 +110,36 @@ def test_positive_validate_katello_certs_check_output(self):
             self.validate_output(result)
 
     @destructive
-    @stubbed()
     def test_positive_update_katello_certs(self):
         """Update certificates on a currently running satellite instance
 
         :id: 0ddf6954-dc83-435e-b156-b567b877c2a5
 
         :steps:
 
-            1. Generate custom certs
-            2. Run katello-certs-check with the required valid arguments
+            1. Use Jenkins provided custom certs
+            2. Assert hammer ping reports running Satellite
             3. Update satellite with custom certs
-            4. Assert output is having correct capsule-certs-generate commands
+            4. Assert output does not report SSL certificate error
+
 
         :expectedresults: Katello-certs should be updated.
 
         :CaseAutomation: notautomated
         """
+        with get_connection(timeout=600) as connection:
+            result = connection.run('hammer ping')
+            assert 'SSL certificate verification failed' not in result.stdout
+            assert 'ok' in result.stdout[1]
+            result = connection.run(
+                'satellite-installer --scenario satellite '
+                '--certs-server-cert /tmp/server.valid.crt '
+                '--certs-server-key /tmp/server.key '
+                '--certs-server-ca-cert  /tmp/rootCA.pem '
+                '--certs-update-server --certs-update-server-ca')
+            result = connection.run('hammer ping')
+            assert 'SSL certificate verification failed' not in result.stdout
+            assert 'ok' in result.stdout[1]
```

Comment:
It would be great to check whether all services are running after certs are updated

Reply:
OK, done. thank you.